include/CL/sycl/program.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef TRISYCL_SYCL_PROGRAM_HPP
+#define TRISYCL_SYCL_PROGRAM_HPP
+
+/** \file The OpenCL SYCL program
+
+    Ronan at Keryell point FR
```

Comment:
The local tradition is to add the name of the initial author of the file. So add your name in the files you created, not mine. :-)